src/main/java/gov/nasa/pds/tools/validate/rule/pds4/TableDataContentValidationRule.java

```diff
@@ -133,11 +174,136 @@ public boolean isApplicable(String location) {
     }
     return isApplicable;
   }
+
+  private void validateAsciiStringFieldsFormat(List<String> fieldFormatList, List<String> fieldTypeList, List<String> fieldNumberList) {
+    // ASCII String related fields should not have '+' in the format.
+
+    int indexToList = 0;
+    // Loop through all elements fieldFormatList.  If field is in ASCII_STRING_TYPE_LIST, check if prohibited '+' is in the format.
+    for (String oneFieldFormat : fieldFormatList) {
+        //LOG.debug("validateAsciiStringFieldsFormat:indexToList,oneFieldFormat,fieldTypeList.get(indexToList)  {},{},{}",indexToList,oneFieldFormat,fieldTypeList.get(indexToList));
+        // Check for prohibited plus symbol '+' in format field : %+8s
+        if (this.ASCII_STRING_TYPE_LIST.contains(fieldTypeList.get(indexToList).toUpperCase())) {
+            if (oneFieldFormat.contains("%+")) {
+                LOG.error("ASCII String related fields should not contain '+' symbol in field_format [" + oneFieldFormat + "] in field_number " + fieldNumberList.get(indexToList));
+                addTableProblem(ExceptionType.ERROR, 
+                    ProblemType.TABLE_FILE_READ_ERROR,
+                    "ASCII String related fields should not contain '+' symbol in field_format [" + oneFieldFormat + "] in field_number " + fieldNumberList.get(indexToList),
+                    getTarget(),
+                    -1,
+                    -1);
+            } 
+       }
+       indexToList += 1;
+    }
+  }
+
+  private void validateAsciiNumberFieldsFormat(List<String> fieldFormatList, List<String> fieldTypeList, List<String> fieldNumberList) {
+    // ASCII Number related fields should not have '-' in the format.
+
+    int indexToList = 0;
+    // Loop through all elements fieldFormatList.  If field is in ASCII_NUMBER_TYPE_LIST, check if prohibited '-' is in the format.
+    for (String oneFieldFormat : fieldFormatList) {
+        //LOG.debug("validateAsciiNumberFieldsFormat:indexToList,oneFieldFormat,fieldTypeList.get(indexToList)  {},{},{}",indexToList,oneFieldFormat,fieldTypeList.get(indexToList));
+        // Check for prohibited minus symbol '-' in format field : %-2d
+        if (this.ASCII_NUMBER_TYPE_LIST.contains(fieldTypeList.get(indexToList).toUpperCase())) {
+            if (oneFieldFormat.contains("%-")) {
+                LOG.error("ASCII Number related fields should not contain '-' symbol in field_format [" + oneFieldFormat + "] in field_number " + fieldNumberList.get(indexToList));
+                addTableProblem(ExceptionType.ERROR, 
+                    ProblemType.TABLE_FILE_READ_ERROR,
```

Comment:
I think we should probably use the `ProblemType.INVALID_LABEL` since this is really a bad label specification, not the data itself. does that seem reasonable?